.eslintrc.js

```diff
@@ -4,10 +4,53 @@ module.exports = {
     ecmaVersion: 2017,
     sourceType: 'module'
   },
-  extends: 'eslint:recommended',
+  plugins: [
+    'ember'
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended'
+  ],
   env: {
     browser: true
   },
   rules: {
-  }
+  },
+  overrides: [
```

Comment:
Not sure why add this override

Reply:
Like the PR says, just walking through what `ember init` does with each subsequent version

Reply:
Sorry! I haven't looked it well! Thanks a lot!